src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.birth();
+  }
+
+  birth() {
+    Animal.alive.push(this);
```

Comment:
it's more convenient to make methods that work with `static` fields, in fact, `static`